THIRDPARTY

```diff
@@ -0,0 +1,67 @@
+Mathesar contains derivatives of the following components that are subject to 
+separate license terms. Your use of these derivatives is subject to the separate
+license terms applicable to each component, in addition to Mathesar's license.
+The full text of all referenced licenses is appended at the end of this file.
```

Comment:
I think it's better to have a `LICENSES` folder where we copy the actual `LICENSE` file from the project that includes the copyright information.

Reply:
I think that would best meet this clause from the MIT license.

> The above copyright notice and this permission notice (including the next
> paragraph) shall be included in all copies or substantial portions of the
> Software.

Reply:
Changes made in https://github.com/centerofci/mathesar/pull/363/commits/2e798b12b54db99d2857264ba3680d19ad44f23d